app/src/main/java/org/mozilla/fenix/home/HomeFragment.kt

```diff
@@ -534,11 +526,12 @@ class HomeFragment : Fragment() {
 
         if (bundleArgs.getBoolean(FOCUS_ON_ADDRESS_BAR)) {
             navigateToSearch()
-        } else if (bundleArgs.getLong(FOCUS_ON_COLLECTION, -1) >= 0) {
-            /* Triggered when the user has added a tab to a collection and has tapped
-            * the View action on the [TabsTrayDialogFragment] snackbar.*/
-            scrollAndAnimateCollection(bundleArgs.getLong(FOCUS_ON_COLLECTION, -1))
         }
+        // else if (bundleArgs.getLong(FOCUS_ON_COLLECTION, -1) >= 0) {
```

Comment:
Should this just be deleted?

Reply:
Yes, the functionality that this would call was dropped. Thank you!